src/catalog/src/builtin.rs

```diff
@@ -2842,7 +2842,7 @@ pub static MZ_SINK_STATISTICS: Lazy<BuiltinSource> = Lazy::new(|| BuiltinSource
         .with_column("messages_committed", ScalarType::UInt64.nullable(false))
         .with_column("bytes_staged", ScalarType::UInt64.nullable(false))
         .with_column("bytes_committed", ScalarType::UInt64.nullable(false)),
-    is_retained_metrics_object: false,
+    is_retained_metrics_object: true,
```

Comment:
this may not be required, as its set on the index, but i set it here for consistency with other retained views

Reply:
I believe it is required. Otherwise we'd lose the historical data on restart.

Reply:
nice!